src/mm-rating/mm-rating.js

```diff
@@ -0,0 +1,113 @@
+( function (scope) {
+    scope.Rating = Polymer({
+        is: 'mm-rating',
+        hovered: false,
+        properties: {
+            icon: {
+                type: String,
+                value: 'star'
+            },
+            customIcon: {
+                type: String,
+                value: null
+            },
+            iconWidth: {
+                type: Number,
+                value: 20
+            },
+            iconHeight: {
+                type: Number,
+                value: 20
+            },
+            rating: {
+                type: Number,
+                value: 0
+            },
+            displayTotals: {
+                type: Boolean,
+                value: false
+            },
+            displayGreetings: {
+                type: Boolean,
+                value: false
+            },
+            readonly: {
+                type: Boolean,
+                value: false
+            },
+            values: {
+                type: Object,
+                value: {0: 'Poor', 1: 'Need Improvement', 2: 'Reasonable', 3: 'Good', 4: 'Excellent'}
```

Comment:
for default object values you should use a function to return the initial value.

`value: function() { return {0:'Poor' }; } //etc`